include/clap/host.h

```diff
@@ -35,6 +35,12 @@ typedef struct clap_host {
    void(CLAP_ABI *request_process)(const struct clap_host *host);
 
    // Request the host to schedule a call to plugin->on_main_thread(plugin) on the main thread.
+   // This callback should be called as soon as practicable, usually in the host application's next
+   // available main thread time slice. Typically callbacks occur at least in the 10s-to-50s-of-milliseconds
+   // or 30-120hz timeslice range. Plugins should not make assumptions about the exactness of timing for
+   // a main thread callback but hosts should endeavour to be prompt. However in high load situations
+   // the host may starve the gui/main thread in favor of audio processing, leading to substantially
+   // longer latencies for the callback than the indicative times given here.
```

Comment:
Note: for perfect 120Hz or 240Hz GUI FPS, the plugin should consider rendering the GUI in a dedicated thread.